examples/contrib/readhar.py

```diff
@@ -0,0 +1,134 @@
+"""Reads HAR files into flow objects"""
+import asyncio
+import codecs
+import json
+import logging
+import time
+
+from mitmproxy import command
+from mitmproxy import connection
+from mitmproxy import ctx
+from mitmproxy import exceptions
+from mitmproxy import http
+from mitmproxy import types
+
+
+class ReadHar:
+    def __init__(self):
+        self.harFile = {}
+        self.flows = []
+
+    def fix_headers(self, request_headers):
```

Comment:
Please add type annotations for all methods. Those are super important to catch errors early. :)